docusaurus/docs/Android/02-client/06-guides/08-optimizations.md

```diff
@@ -0,0 +1,25 @@
+# Optimizations
+
+There are some optimizations in the SDK to avoid waste of resouces and improve performance. It is possible to configure it to disable or enabled then.
```

Comment:
```suggestion
The SDK is optimized to avoid waste of resources and improve performance. You can freely choose which optimizations should be enabled or disabled.
```